core/flyout_horizontal.ts

```diff
@@ -285,7 +285,8 @@ export class HorizontalFlyout extends Flyout {
         } else {
           moveX = cursorX - tab;
```

Comment:
I don't think we actually need this `moveX` variable, but I'm not gonna ask you to change it! Just leaving a note for my future self.

Reply:
I did try to do the testing manually, but couldn't get completely was the correct way of doing it or not?

Reply:
You can follow the reproduction steps from the [original issue](https://github.com/google/blockly/issues/6280#issue-1315272059) to see if it reproduces the bug or not =)

In your case you can go into the [playground.html](https://github.com/google/blockly/blob/develop/tests/playground.html) and change `getToolboxElement` to the following:

```js
    function getToolboxElement() {
        return {
          "kind": "flyoutToolbox",
          "contents": [
            {
              "kind": "block",
              "type": "controls_if",
              "x": 10,
              "y": 10
            },
            {
              "kind": "block",
              "type": "controls_whileUntil",
              "x": 30,
              "y": 30
            }
          ]
        };
      }
```

The "x" and "y" properties were what was problems before. So these should be ignored and you should see blocks that are aligned correctly =)

If you could take a screen shot to confirm how it looks that would be perfect!

Reply:
I have updated the PR remark regarding the testing, and the required SS are attached below in the comment.